resources/idempotence_file.rb

```diff
@@ -2,6 +2,7 @@
 
 resource_name :idempotence_file
 provides :idempotence_file
+unified_mode true
```

Comment:
This is kind of the whole reason for the PR. It causes a spammy warning in Chef 17.